src/hooks/useIsProfileValidatedOnDiscord.ts

```diff
@@ -0,0 +1,11 @@
+import { AccountType } from '../entities/User/types'
+
+import useIsProfileValidated from './useIsProfileValidated'
+
+function useIsProfileValidatedOnDiscord(address: string | null) {
+  const { isProfileValidated, validationChecked } = useIsProfileValidated(address, [AccountType.Discord])
```

Comment:
Instead of creating a new hook I'd use `useIsProfileValidated(address, [AccountType.Discord]`